### PR TITLE
Plot F2 score, precision, recall, and common false negative/positive tokens

### PR DESCRIPTION
**Pull request by benkilimnik**
Add `Plotter` class to presidio `Evaluator` to enable graphing of F2 score, precision, recall, and common false positive/negative tokens. 

Example usage:
```
evaluator = Evaluator(model=model)
evaluation_results = evaluator.evaluate_all(dataset)
results = evaluator.calculate_score(evaluation_results)

error_analyzer = Evaluator.Plotter(model, results, output_folder, model_name, beta=2.5)
error_analyzer.plot_scores()
error_analyzer.plot_most_common_tokens()
```

Example plots generated with this class: 

presidio_most_common_fn

f2_scores_presidio_spacy

### CODE REVIEW

**Comment by omri374** [comment]
Thanks @benkilmnit! How important is `kaleido` for this functionality? It's under MIT license but also encapsulates chromium which comes with various other licenses.

File: presidio_evaluator/evaluation/evaluator.py [hunk 1]
```diff
@@ -344,3 +347,119 @@ def f_beta(precision: float, recall: float, beta: float) -> float:
...
+            """
+            Plots per-entity recall, precision, or F2 score for evaluated model. 
+            :param plot_type: which metric to graph (default is F2 score)
+            """
+            scores = {}
+            scores['entity'] = list(self.results.entity_recall_dict.keys())
+            scores['recall'] = list(self.results.entity_recall_dict.values())
+            scores['precision'] = list(self.results.entity_precision_dict.values())
+            scores['count'] = list(self.results.n_dict.values())
+            scores['f2_score'] = [Evaluator.f_beta(precision=precision, recall=recall, beta=2.5)
```
Comment (omri374) [inline presidio_evaluator/evaluation/evaluator.py:379]: Consider having `beta` as a parameter to this class
```suggestion
            scores[f"f{self.beta}_score"] = [Evaluator.f_beta(precision=precision, recall=recall, beta=self.beta)
```

Comment (benkilimnik) [reply presidio_evaluator/evaluation/evaluator.py]: Done

**Comment by benkilimnik** [comment]
`kaleido` is used only to save the graphs to png. I've just removed it.

File: presidio_evaluator/evaluation/evaluator.py [hunk 2]
```diff
@@ -344,3 +348,116 @@ def f_beta(precision: float, recall: float, beta: float) -> float:
...
+
+    class Plotter:
+        """
+        Plot scores (f2, precision, recall) and errors (false-positivies, false-negatives) 
+        for a PII detection model evaluated via Evaluator
+
+        :param model: Instance of a fitted model (of base type BaseModel)
+        :param results: results given by evaluator.calculate_score(evaluation_results)
+        :param output_folder: folder to store plots and errors in
+        :param model_name: name of the model to be used in the plot title
```
Comment (omri374) [inline presidio_evaluator/evaluation/evaluator.py:360]: Could you please add the missing params to the docstring?

**Review by omri374 — APPROVED** [review]

**Pull request merged**